src/maxent_params.cpp

```diff
@@ -240,7 +240,7 @@ void ContiParameters::decompose_covariance_matrix(const alps::params& p){
     // below that value)
     int new_ndat_,old_ndat_=ndat();
     for (new_ndat_ =0;new_ndat_<ndat();new_ndat_++)
-      if (var[new_ndat_]>1e-10) break;
+      if (var[new_ndat_]>1e-12) break; //pick cutoff value manually after looking at evalue spectrum
```

Comment:
Please leave the default at 1.e-10 as it was, but insert a parameter that can change this to 1.e-12 (or any other value) as desired by a user.